exporters/trace/jaeger/jaeger_test.go

```diff
@@ -330,7 +330,7 @@ func TestExporter_ExportSpan(t *testing.T) {
 	assert.True(t, span.SpanContext().IsValid())
 
 	exp.Flush()
-	tc := exp.uploader.(*testCollectorEnpoint)
+	tc := exp.uploader.(*testCollectorEndpoint)
 	assert.True(t, len(tc.batchesUploaded) == 1)
```

Comment:
I think we need to check more details of the batches.

- The process from the batch
  - Service name
  - Tags

Perhaps we also need to create multiple spans to see if the exporter can merge these spans into a single batch correctly.

Reply:
Do you mean adding assertions in `TestExporter_ExportSpan` just like I did in `TestNewExporterPipelineWithOptions`?

Reply:
@IrisTuntun: yeah, something like what you did in the other test case, but make sure to check the things @XSAM listed  (i.e. also the service name).